ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -487,3 +487,14 @@ def norm(tensor, ord="euclidean", axis=None, keepdims=None, name=None):
     "float32",
     "float64",
 )
+
+
+@to_ivy_arrays_and_back
+def unique(x, out_idx=ivy.int32, name=None):
+    y, idx = [], []
+    for i in x:
+        if i not in y:
+            y.append(i)
+        idx.append(y.index(i))
+    y, idx = ivy.array(y), ivy.astype(idx, out_idx)
+    return y, idx
```

Comment:
There is a `ivy.unique_all` function you should be using. It will be more efficient than manually finding the unique values.

Reply:
I tried using `ivy.unique_all` but `tf.unique` has a different behavior, according to the docs. E.g:

```
x = np.array([2,1,1,1,1,1,1,1,1,1])
res = ivy.unique_all(x)

for i in res: print(i)

>> ivy.array([1, 2])
>> ivy.array([1, 0])
>> ivy.array([1, 0, 0, 0, 0, 0, 0, 0, 0, 0])
>> ivy.array([9, 1])
```

The output for `tf.unique` for the same input array is:
```
x = tf.constant([2,1,1,1,1,1,1,1,1,1])
res = tf.unique(x)

for i in res: print(i)

>> tf.Tensor([2 1], shape=(2,), dtype=int32)
>> tf.Tensor([0 1 1 1 1 1 1 1 1 1], shape=(10,), dtype=int32)
```

Reply:
The difference between `ivy.unique_all` and `tf.unique` was because tensorflow sorts the unique elements in the order of occurrence while ivy sorted them by value. This was something we had overlooked originally when writing `ivy.unique_all`. I have just added a new argument to `ivy.unique_all` that allows one to choose the ordering. Feel free to pull the master and use the updated function here.

Reply:
I've implemented the required changes.